tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/GivenUsingSuite.scala

```diff
@@ -105,10 +108,10 @@ class GivenUsingSuite extends BaseDottySuite {
     )
   }
 
```

Comment:
(random place)
with can start in that case indentation block, could you add a test for that?
sth like:
```
given intOrd: Ord[Int] with
   def fa: Int = 1
   def fb: Int = 2
```

Reply:
Actually it was not supported, but it's not that hard to do. Fixed it and added a test for both indentation and legit `with`.